esp/services/ws_logaccess/WsLogAccessService.cpp

```diff
@@ -13,11 +13,85 @@ bool Cws_logaccessEx::onGetLogAccessInfo(IEspContext &context, IEspGetLogAccessI
     bool success = true;
     if (queryRemoteLogAccessor())
     {
+        IPropertyTree * logColumnDefinitions = queryRemoteLogAccessor()->queryLogMap();
```

Comment:
trivial: to be consistent with line 11, remove the space from the front of the '*'.